core/src/main/java/eu/fasten/core/data/opal/MavenCoordinate.java

```diff
@@ -156,9 +156,15 @@ public String toProductUrl() {
         var repo = mavenRepos.get(0);
         return this.toURL(repo) + "/" + artifactID + "-" + versionConstraint + "." + packaging;
```

Comment:
This has not been changed, but does not make sense. Which URL is this supposed to create?

Reply:
The method `toProductURL()` is not written by me. Based on its usage, I can tell it is used to download Maven artifacts like JARs.